src/api/UserApi.tsx

```diff
@@ -28,6 +28,17 @@ const put = async (data: any) => {
 
     return response;
 };
+
+const getUserProfileAccess = async (userid: string, focusUserId: string) => {
+    const response = await axios.get(`${`${BASE_URL}UserAccess/GetUserProfileAccess/${userid}/${focusUserId}`}`)
+    .catch(error => {
+        throw error;
+    });
+        
+    
```

Comment:
Favorite issue of all times - do we need this white-space?

Reply:
Favorite answer — we don't need this white-space :)